src/Components/InputField/NotesField.jsx

```diff
@@ -20,6 +39,7 @@ const NotesField = ({ notes, setNotes, addNoteHandler, updatehandler }) => {
   return (
     <>
       <form
+        style={{ background: noteBgColor }}
         className="notes-field-wrapper "
         onSubmit={
           !editNote
```

Comment:
onSubmit function seems to be missing

Reply:
I have added onSubmit functionality can you check correctly.
so, based on the ```editNote``` I'm adding a function